lib/devicetrust/testenv/fake_device_service.go

```diff
@@ -483,6 +484,58 @@ func enrollMacOS(stream devicepb.DeviceTrustService_EnrollDeviceServer, initReq
 	}, ecPubKey, nil
 }
 
+// CreateAssertCeremony creates a fake, server-side device assertion ceremony.
+func (s *FakeDeviceService) CreateAssertCeremony() (assertserver.Ceremony, error) {
+	return s, nil
+}
+
+// AssertDevice implements a fake, server-side device assertion ceremony.
+//
+// AssertDevice requires an enrolled device, so the challenge signature
+// can be verified.
+func (s *FakeDeviceService) AssertDevice(ctx context.Context, stream assertserver.AssertDeviceServerStream) (*devicepb.Device, error) {
```

Comment:
This is great, thanks.

We could probably simplify [assert_test.go][1] using this.

[1]: https://github.com/gravitational/teleport/blob/dbe1295cdcf24325ba37be505c51e0b4a8248fa9/lib/devicetrust/assert/assert_test.go#L96